editoast/src/schema/mod.rs

```diff
@@ -98,6 +99,22 @@ impl ObjectType {
             ObjectType::Catenary => "osrd_infra_catenarymodel",
         }
     }
+
+    pub fn get_layer_table(&self) -> &'static str {
+        match *self {
+            ObjectType::TrackSection => "osrd_infra_tracksectionlayer",
+            ObjectType::Signal => "osrd_infra_signallayer",
+            ObjectType::SpeedSection => "osrd_infra_speedsectionlayer",
+            ObjectType::Detector => "osrd_infra_detectormodel",
+            ObjectType::TrackSectionLink => "osrd_infra_tracksectionlinklayer",
+            ObjectType::Switch => "osrd_infra_switchlayer",
+            ObjectType::SwitchType => "osrd_infra_switchtypelayer",
```

Comment:
This layer doesn't exists since switch type doesn't have geometry. We should exclude switch type from this endpoint.

-----

This function can return an error if `ObjectType::SwitchType` is given.